x-pack/plugins/ml/public/embeddables/anomaly_swimlane/anomaly_swimlane_embeddable.tsx

```diff
@@ -82,16 +95,13 @@ export class AnomalySwimlaneEmbeddable extends Embeddable<
 
   constructor(
     initialInput: AnomalySwimlaneEmbeddableInput,
-    private services: [CoreStart, MlStartDependencies, AnomalySwimlaneServices],
+    public services: [CoreStart, MlDependencies, AnomalySwimlaneServices],
     parent?: IContainer
   ) {
     super(
       initialInput,
       {
-        jobIds: initialInput.jobIds,
-        swimlaneType: initialInput.swimlaneType,
         defaultTitle: initialInput.title,
```

Comment:
I don't seem to be able to edit the embeddable panel title, when I add a new embeddable, or edit an existing one. Do you see the same?

![image](https://user-images.githubusercontent.com/7405507/87297679-ba084f00-c500-11ea-8e60-5804b803702c.png)

Reply:
I'm able to edit the title, but there was an issue with a swim lane initializer. Fixed in https://github.com/elastic/kibana/pull/71082/commits/5c39b47ef5c96ce0d8357f5ea033e3ea2a0e82cf

Reply:
I can edit it now, but only by keyboard navigation into the text box. But this affects all the visualizations on my dashboard; not just the swim lane embeddable.

Reply:
You might need to run `yarn kbn clean` and re-bootstrap. Changing the title works for me for all panels on the dashboard.